lib/powerpoint/presentation.rb

```diff
@@ -23,21 +23,65 @@ def add_intro(title, subtitile = nil)
     end
 
     def add_textual_slide(title, content = [])
-      @slides << Powerpoint::Slide::Textual.new(presentation: self, title: title, content: content)
+      slide = Powerpoint::Slide::DefaultSlide.new({
+          title: title,
+          slide_layout: 2,
+          elements: [
+          Powerpoint::Slide::TextContent.new(idx: 1, bullets: true, content: content)
+      ]})
+
+      @slides << slide
     end
 
     def add_pictorial_slide(title, image_path, coords = {})
-      @slides << Powerpoint::Slide::Pictorial.new(presentation: self, title: title, image_path: image_path, coords: coords)
+
+      img_coords = { x: 826325, y: 1825625, cx: 10515600, cy: 4351338 }
+
+      slide = Powerpoint::Slide::DefaultSlide.new({
+        title: title,
+        slide_layout: 2,
+        elements: [
+          Powerpoint::Slide::ImageContent.new(idx: 1, image_path: image_path, centered: true, coords: img_coords)
+      ]})
+
+      @slides << slide
     end
 
     def add_text_picture_slide(title, image_path, content = [])
-      @slides << Powerpoint::Slide::TextPicSplit.new(presentation: self, title: title, image_path: image_path, content: content)
+
+      img_coords = { x: 6172200, y: 1825625, cx: 5181600, cy: 4351338 }
```

Comment:
magic numbers coming from the image coordinates in the template file :/ if the templates were explicit datastructures I'd like to just grab them ... but for right now they're not